styles.css

```diff
@@ -610,3 +660,491 @@ li {
+    font-style: normal;
+    font-weight: 700;
+    font-size: 40px;
+    line-height: 52px;
+    padding: 0;
+    margin: 0;
+  }
+
+  .card {
+    width: 380px;
+    height: 498px;
+    border: 1px solid #d0d9d4;
+    border-radius: 8px;
+    background: #ebf0ee;
+  }
+
+  .card_inside {
+    width: 380px;
+    height: 238px;
+    align-items: center;
+    position: relative;
+    top: 250px;
+    background: #fff;
+    border-radius: 8px;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    text-align: center;
+    align-content: center;
+  }
+
+  .paragraph_card {
+    width: 337px;
+    height: 57px;
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    font-weight: 600;
+    font-size: 22px;
+    line-height: 32px;
+    color: #172b4d;
+    padding: 10px;
+    position: relative;
+    margin: 0;
+    padding-top: 45px;
+  }
+
+  .stack {
+    width: 103px;
+    height: 32px;
+    font-family: 'Inter', sans-serif;
+    font-weight: 12px;
+    line-height: 16px;
+    letter-spacing: 0.03em;
+    color: #172b4d;
+    display: flex;
+    align-items: center;
+    gap: 4px;
+    position: relative;
+    right: 36%;
+    top: 10px;
+    justify-content: space-between;
+  }
+
+  .listw {
+    width: 103px;
+    height: 32px;
+    font-family: 'Inter', sans-serif;
+    font-weight: 12px;
+    line-height: 16px;
+    letter-spacing: 0.03em;
+    color: #172b4d;
+    display: flex;
+    align-items: center;
+    gap: 10px;
+    justify-content: space-between;
+  }
+
+  .listw li {
+    padding: 8px 12px;
+    background: #ebf0ee;
+    border-radius: 4px;
+    position: relative;
+    left: 1px;
+  }
+
+  #button {
+    position: relative;
+    left: 1%;
+  }
+
+  /* About section */
+
+  .images_about {
+    display: contents;
+  }
+
+  .img_about1 {
+    position: absolute;
+    width: 288.04px;
+    height: 140.24px;
+    bottom: 55%;
+    left: -13%;
+  }
+
+  .img_about2 {
+    position: absolute;
+    width: 269.55px;
+    height: 244.78px;
+    bottom: 60%;
+    right: -13%;
+  }
+
+  .about {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+    padding: 80px 0;
+    gap: 48px;
+    position: absolute;
+    width: 1171px;
+    height: 808px;
+    left: 115px;
+    top: 2300px;
+    background: none;
+  }
+
+  .text_about {
+    padding-left: 34%;
+  }
+
+  .text2_about {
+    width: 920px;
+    height: 96px;
+    font-size: 20px;
+    line-height: 32px;
+    padding-top: 15px;
+    position: relative;
+    left: 30px;
+  }
+
+  #button_resume {
+    position: relative;
+    left: 410px;
+    top: -20px;
+  }
+
+  .skills_container {
+    display: inline-grid;
+    align-items: center;
+    padding: 4px 8px;
+    grid-template-columns: auto auto auto;
+    gap: 24px;
+    width: 1171px;
+    height: 360px;
+    padding-top: 0;
+    margin-top: 0;
+    position: absolute;
+    top: 60%;
+    left: 1.5%;
+  }
+
+  .card_skills {
```

Comment:
- [ ] As the previous reviewer mentioned. these issues still persist so kindly consider fixing them as the previous reviewer suggested. 
your design
![gf](https://user-images.githubusercontent.com/96683843/213874719-89f03865-ab5a-47e4-99db-c506313834e9.jpg)
![re](https://user-images.githubusercontent.com/96683843/213874729-d2624786-2863-4a3e-88ab-da031b7391d6.jpg)